docroot/themes/humsci/humsci_basic/src/scss/components/_main-nav.scss

```diff
@@ -1,105 +1,354 @@
-// Override some decanter styles for Main Nav
-// scss-lint:disable ImportantRule
-.su-main-nav {
+.hb-main-nav {
```

Comment:
a note from our earlier conversation: but you had said the last nav item on desktop should have the sub nav right instead of left aligned.